jormungandr/src/notifier/mod.rs

```diff
@@ -72,22 +89,60 @@ impl Notifier {
                             }
                         });
                     }
-                    Message::NewConnection(ws) => {
-                        trace!(logger, "processing notifier new connection");
+                    Message::FragmentRejected(fragment_id) => {
+                        info.spawn("notifier broadcast fragment rejection", async move {
+                            if let Err(_err) =
+                                mempool_sender.send(MemPoolMessage::FragmentRejected(fragment_id))
+                            {
+                                ()
+                            }
```

Comment:
I don't really understand the purpose of the `if let` block here. Can't we use `map_err` or something else?